### ISSUE TITLE
Item P2P Server crash

### ISSUE THREAD

**Issue by PssbleTrngle** [AppliedEnergistics/Applied-Energistics-2#5269]
**Describe the bug**
Crash when loading a server world with P2P Item tunnels

**To Reproduce**
Place an input and output P2P Item tunnel

**Additional context**
```
net.minecraft.crash.ReportedException: Ticking block entity
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:855) ~[?:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:291) ~[?:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:787) ~[?:?]
        at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?]
        at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?]
        at java.lang.Thread.run(Thread.java:829) [?:?]
Caused by: java.lang.NullPointerException
        at appeng.parts.p2p.ItemP2PTunnelPart.access$300(ItemP2PTunnelPart.java:37) ~[appliedenergistics2:8.3.1]
        at appeng.parts.p2p.ItemP2PTunnelPart$OutputItemHandler.getSlots(ItemP2PTunnelPart.java:175) ~[appliedenergistics2:8.3.1]
        at net.minecraftforge.items.VanillaInventoryCodeHooks.lambda$extractHook$0(VanillaInventoryCodeHooks.java:56) ~[forge:?]
        at java.util.Optional.map(Optional.java:265) ~[?:?]
        at net.minecraftforge.items.VanillaInventoryCodeHooks.extractHook(VanillaInventoryCodeHooks.java:53) ~[forge:?]
        at net.minecraft.tileentity.HopperTileEntity.func_145891_a(HopperTileEntity.java:181) ~[?:?]
        at net.minecraft.tileentity.HopperTileEntity.func_200111_K(HopperTileEntity.java:93) ~[?:?]
        at net.minecraft.tileentity.HopperTileEntity.func_200109_a(HopperTileEntity.java:109) ~[?:?]
        at net.minecraft.tileentity.HopperTileEntity.func_73660_a(HopperTileEntity.java:92) ~[?:?]
        at net.minecraft.world.World.func_217391_K(World.java:491) ~[?:?]
        at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:371) ~[?:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:851) ~[?:?]
        ... 5 more
```
[Crash Report](https://gist.github.com/PssbleTrngle/7bbe278f0633e6d98dc4e1b895f08994)

**Environment**
Forge: 1.16.5-36.1.4
AE2: 8.3.1

**Comment by Technici4n** [comment]
Will be fixed by #5251.

**Comment by PssbleTrngle** [comment]
Oh very nice, I did only look at the issue and not on the pull request
